codespell_lib/data/dictionary.txt

```diff
@@ -7823,13 +7824,41 @@ depandent->dependent
 deparment->department
 deparmental->departmental
 deparments->departments
+depden->depend
+depdence->dependence
```

Comment:
It would be worth adding the in- versions of these typos too where they aren't already in the dictionary, e.g. indepdence->independence